public/locales/en/translation.json

```diff
@@ -24,13 +32,30 @@
     },
     "title": "Wallet Balance"
   },
+  "walletModal": {
```

Comment:
don't forget to add all new keys to the Hindi translation file. we don't care about the correct translation right now (you can use Google Translator), but we don't want to have a bunch of missing keys in the future

Reply:
Done.
Also, we have a network name in the Header. And I did not translate it.
"Starknet-Mainnet". I don't know how to properly translate it.
Also, "Argent-X" and "Braavos" remain in English.

Reply:
thanks!
yeah, let's keep them as it